apps/SageAccountWeb/src/routes.tsx

```diff
@@ -0,0 +1,159 @@
+import { createBrowserRouter } from 'react-router'
+import { convertModuleToRouteObject } from '@sage-bionetworks/synapse-portal-framework/utils/convertModuleToRouteObject'
+import AppWrapper from './AppWrapper'
+import App from './App'
+import { RESET_2FA_ROUTE } from './Constants'
+
+export const router = createBrowserRouter([
+  {
+    path: '/',
+    element: (
+      <AppWrapper>
+        <App />
+      </AppWrapper>
+    ),
```

Comment:
These components include logic/context/layout for every page, so there's not really a reason to lazy-load this at this time